Project/Homebrew/README.md

```diff
@@ -0,0 +1,22 @@
+Building
+--------
+
+Install ffmpeg, freetype2, qt5
+
+    $ brew install ffmpeg freetype2 qt5
+
```

Comment:
Oh, I don't remember having to do that but I only have Qt5 on my homebrew. Btw, I will suggest adding a qwt-qt5 to homebrew like Linux distros do because I don't think it's possible to build only a single library that links against two different Qt frameworks. I'll open a pull request there in the following days and see what the maintainers think about it.